internal/indexer/config.go

```diff
@@ -26,3 +26,33 @@ func checkConfigChange(ctx context.Context, storage storage.Storage, cfg config.
 		}
 	}
 }
+
+func fetchConfigurations(cr config.ContractRepository, cfgChan chan<- []config.Config) {
+	for {
+		time.Sleep(5 * time.Second)
+		cfgs, err := cr.GetConfigs()
+		if err != nil {
+			log.Error("failed to get configs", "error", err)
```

Comment:
debug here?

Reply:
no get out of loop?

Reply:
could be single error not necessarily repearting

Reply:
i.e database timeout while server restarting, might not want to exit yet